### DISCUSSION TITLE
上交大出国指南

### DISCUSSION THREAD

**Discussion by FromBei-JingWithLove** [The-Run-Philosophy-Organization/run#141] [General]
在出国的问题上，这山看着那山高的心理是极其可怕的。如果在一个对你的血统没有歧视的国度里，在一个竞争并不算激烈、甚至生活上还称得上有保障的社会中，在一个对你来说没有任何语言、文化障碍的环境下，你都算不上一个成功者的话，你有什么资本可以奢望在出国后，自己能够摇身一变，以胜者的姿态生存下来？你又有什么理由不继续失败？你怎么就那么自信地说因为我在国内做不好，所以我出国就能做好？成功的人在不同的地方有不同的成功方式，但是失败的人在不同的地方会有完全相同的失败方法。

**Comment by CharryLee0426** [comment]
“竞争并不算激烈”是指在美国吗？

**Comment by taobowen** [comment]
出国是为了成功吗？这是什么韭菜思维？

**Comment by Theodoree** [comment]
人不能被意识形态给圈定住了,人应该有选择的权利。
弗里德曼曾经说过:自由人既不会问他的国家能为他做些什么，也不会问他能为他的国家做些什么

**Reply by 2jackylee** [reply]
https://www.sohu.com/a/551376471_120094090?editor=%E6%9C%B1%E9%B9%8F%E8%8B%B1%20UN603&scm=1104.0.0.0&spm=smpc.news-home.top-news3.4.1653558489648s3PjANP&_f=index_chan08news_8      那这位博士生这辈子在国内已经是半废状态了 怎么办呢😅“-1”

**Comment by the-run-philosophy** [comment]
如果你不知道为什么要润的话请参考：[哲学概念](https://github.com/The-Run-Philosophy-Organization/run/tree/main/%E5%93%B2%E5%AD%A6%E6%A6%82%E5%BF%B5) 把他们都搞清楚之后再问问你自己要不要润。

**Comment by code-wd** [comment]
> 

你以为你是谁呀，你写的是什么东西？你真的读过中西方哲学、马克思哲学吗？你写的东西符合逻辑吗？是客观事实吗？如果没有，请不要误人子弟，闭嘴不瞎说对自己负责，更是对别人负责

**Comment by yuchen001** [comment]
围观耍猴

**Comment by PeterWrighten** [comment]
建议这个issue追加进“咱们赢麻了”言论精选 @the-run-philosophy

**Comment by MarikoKujo** [comment]
自己不想润，一些同行润出去了，不是减轻了您在国内的生存压力么？另外“你们这些想润的在等国都是loser”这样居高临下的无端假设，可实在不符合您ID里的“WithLove”呢。
有空去别处拉人来“反制”本项目，不如想想自己作为一个国内的“成功者”，为何还活得这么拧巴，连几篇观念不同的帖子都看不顺眼，嘻嘻👀

**Comment by leaf2030** [comment]
出国是为了不当韭菜 并不是为了成功

**Comment by mitchellcuevas** [comment]
什么傻逼在这里大放厥词。

**Reply by leon-fudan** [reply]
居然连厥词都不会放？

**Comment by tonylyf407** [comment]
管那么多干什么，让他们润出去了对其他中国人有好处

**Comment by MaybeShewill-CV** [comment]
能run的抓紧点run 越快越好 能抓多紧抓多紧：）

**Comment by MarionNoel** [comment]
润太祖可是特大人命事故入狱的局长公子，贪二代，有钱的大老板。他大发善心，关爱新一代it耗材，安排咱们世世代代跟着他，给他做贡献。
https://github.com/MarionNoel/juzhanggongzi/discussions/
